kubernetes-model-generator/generateModel.sh

```diff
@@ -74,6 +88,7 @@ echo "Installing required common modules"
 mvn clean install -f ../pom.xml -N
 mvn clean install -N
 mvn clean install -pl kubernetes-model-common -pl kubernetes-model-jsonschema2pojo
+cd ../model-annotator && mvn clean install && cd -
```

Comment:
Could you use `mvn clean install -f ../model-annotator/pom.xml` for consistency?